internal/command/run/run.go

```diff
@@ -19,7 +21,35 @@ type Command struct {
 	Port string
 }
 
+func frontEnd() string {
+	// TODO: Change to the corresponding front end in next releases
+	return "Svelte " + version.Svelte
+}
+
+func displayDashboard(host, port string, timeElapsed time.Duration) {
+	/*
+		The dashboard should looks something like this:
+			|   bud dev server is running:
+			|
+			| > Listening on: http://127.0.0.1:3000
+			| > Front end: Svelte 3.47.0
```

Comment:
I'd also exclude the Svelte version. The question I'm asking myself is:

"Does the developer need to make a decision with this information right now?"

I'm concluding no at the moment, but could be convinced otherwise.